nullaway/src/test/java/com/uber/nullaway/NullAwayAutoSuggestTest.java

```diff
@@ -102,13 +103,44 @@ public void suggestCastToNonNull() throws IOException {
             "import static com.uber.nullaway.testdata.Util.castToNonNull;",
             "import javax.annotation.Nullable;",
             "class Test {",
-            "  Object test1(@Nullable Object o) {",
+            "  @Nullable Object o;",
+            "  Object test1() {",
             "    return castToNonNull(o);",
             "  }",
             "}")
         .doTest();
   }
 
+  @Test
+  public void suppressInsteadOfCastToNonNull() throws IOException {
```

Comment:
Minor comment noting which cases these are and referencing the ErrorBuilder docs? Just because when not seen as part of the PR it might be hard to remember why we want this behavior.

Also, can we add a case where `null` is assigned to a non-null field?

Reply:
Fixed in https://github.com/uber/NullAway/pull/799/commits/b84a4f76c595c6e30903ed04db79ddf945a9d4e8